TrainingApp/StartViewController.swift

```diff
@@ -14,15 +14,23 @@ class StartViewController: UIViewController {
     @IBOutlet private var signInButton: UIButton!
     @IBOutlet private var signUpButton: UIButton!
     
+    let userDefaults = UserDefaults.standard
     // MARK: - Lifecycle
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         self.navigationController?.navigationBar.isHidden = false
+        checkLanguage()
```

Comment:
why we use checkLanguage() in viewWillAppear and viewDidLoad ?

Reply:
Localization will not work at app launch without calling checkLanguage() in viewDidLoad.